src/types/DataEntry.ts

```diff
@@ -1,10 +1,13 @@
 export default class DataEntry {
     time?: Date;
-    power1?: number;
-    power2?: number;
+    power?: number;
```

Comment:
`power` (and maybe `time`) could be made non-optional as we only create the object when a `power` is present. We could make both of them `readonly`.